src/components/Button/Button.tsx

```diff
@@ -25,6 +27,8 @@ const Button: FC<Props> = ({ children, className, variant = Variant.Default, typ
       'py-1',
       variant === Variant.Default && ['bg-background', 'text-white'],
       variant === Variant.Primary && ['bg-background', 'text-primary'],
+      variant === Variant.BlueBackground && ['bg-primary', 'text-white', 'w-full', 'border', 'border-primary', 'py-4', 'font-bold', 'text-base', 'text-xl', 'leading-6'],
+      variant === Variant.WhiteBackground && ['bg-white', 'text-primary', 'w-full', 'border', 'border-primary', 'py-4', 'font-bold', 'text-base', 'text-xl', 'leading-6'],
```

Comment:
```suggestion
      variant === Variant.BlueBackground && ['bg-primary', 'text-white'],
      variant === Variant.WhiteBackground && ['bg-white', 'text-primary']
```
布局paddinh width leadibg可以在Button 和基础的font相关的，可以在外面包裹div上添加，写在这里的话，这个Variant的可复用性就大大降低。 border 可以通过 isBorder 的判断传进来来控制是否选择显示border，border color可以通过children 进来。